### PR TITLE
Added HTML login page

### LINKED ISSUE #9: Adding Login Page
For now just a basic HTML page
@orzionpour

### PR DESCRIPTION
**Pull request by orzionpour**
[-] Created static folder - will contain any 'static' related files such as images, videos, text files, etc..
[-] Add falcon logo to static folder.
[-] Created templates folder.
[-] Created login.html file - display falcon's initial login form.
[-] Change layout using css
[-] Add CSS design
Resolve #9.

### CODE REVIEW

**Comment by orzionpour** [comment]
Sorry for opening a new PR, the previous had a few commits which I couldn't squash for some reason.
@kobihk @nunnatsa @mgold1234 @avraham1992 @AvivLiberman @aradz1997

**Review by AvivLiberman — APPROVED** [review]

**Review by AvivLiberman — APPROVED** [review]

**Review by aradz1997 — APPROVED** [review]

**Comment by mgold1234** [comment]
> [-] Created static folder - will contain any 'static' related files such as images, videos, text files, etc.. [-] Add falcon logo to static folder. [-] Created templates folder. [-] Created login.html file - display falcon's initial login form. [-] Change layout using css [-] Add CSS design Resolve #9.

**Comment by orzionpour** [comment]
why it was closed without being merged?

**Comment by mgold1234** [comment]
> [-] Created static folder - will contain any 'static' related files such as images, videos, text files, etc..
> [-] Add falcon logo to static folder.
> [-] Created templates folder.
> [-] Created login.html file - display falcon's initial login form.
> [-] Change layout using css
> [-] Add CSS design
> Resolve #9.

Please be consistent with the timing of the sentences.

**Comment by mgold1234** [comment]
I closed it by mistake:(

**Comment by orzionpour** [comment]
Done, but no success with squashing these commits.. any advice ?

File: falcon/urls.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
+from falcon import views
```
[the remark this answers is not in the document]
Comment (mgold1234) [reply falcon/urls.py:18]: this one?

File: falcon/urls.py [hunk 2]
```diff
@@ -15,7 +15,10 @@
...
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.login ,name='login')
```
[the remark this answers is not in the document]
Comment (mgold1234) [reply falcon/urls.py:23]: ??

**Review by Asafh7 — APPROVED** [review]

**Review by mgold1234 — COMMENTED** [review]
lgtm

**Comment by orzionpour** [comment]
Done with commit squashing.

**Review by natifridman — APPROVED** [review]

**Pull request merged**